src/index.html

```diff
@@ -5,8 +5,360 @@
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__rate">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__rewiews">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">Price</p>
+            <p class="card__price-amount">$2,199</p>
+          </div>
+
+          <a
+          href="#"
+          class="card__buy-button"
+          >
+          Buy
+          </a>
+        </div>
+        <div class="card__item">
+          <img
+            class="card__img"
+            src="../src/images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+          >
+          <h2 class="card__text">
+            APPLE A1419 iMac 27" Retina
+            <br>
+            5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__rate">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__rewiews">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">Price</p>
+            <p class="card__price-amount">$2,199</p>
+          </div>
+
+          <a
+          href="#"
+          class="card__buy-button"
+          >
+          Buy
+          </a>
+        </div>
+        <div class="card__item">
+          <img
+            class="card__img"
+            src="../src/images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+          >
+          <h2 class="card__text">
+            APPLE A1419 iMac 27" Retina
+            <br>
+            5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__rate">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__rewiews">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">Price</p>
+            <p class="card__price-amount">$2,199</p>
+          </div>
+
+          <a
+          href="#"
+          class="card__buy-button"
+          >
+          Buy
+          </a>
```

Comment:
```suggestion
          <a
            href="#"
            class="card__buy-button"
          >
            Buy
          </a>
```
fix all similar cases